### ISSUE TITLE
Fn::Join not accepting a Ref to a CommaDelimitedList as an array argument

### ISSUE THREAD

**Issue by grams** [shalupov/idea-cloudformation#84]
Given this YAML template:

```YAML
---
AWSTemplateFormatVersion: '2010-09-09'
Description: "Sample !Join and !Ref usage"
Parameters:
  aliasesParameter:
    Type: CommaDelimitedList
    Description: Comma separated list of domain names
Resources:
  MyDistribution:
    Type: AWS::CloudFront::Distribution
    Properties:
      DistributionConfig:
        Comment: !Join [ ", ", !Ref aliasesParameter ]
```

The expression `!Join [ ", ", !Ref aliasesParameter ]` is highlighted as an error "Join expects a string argument and an array argument". However, the reference to a CommaDelimitedList is, indeed, an array (of strings), AFAICT.

**Comment by shalupov** [comment]
Fixed in 0.5.26, thanks

**Comment by grams** [comment]
Works like a charm!

Thanks to you for the fix and for this very useful plugin!
